graylog2-server/src/main/java/org/graylog2/alerts/AbstractAlertCondition.java

```diff
@@ -51,15 +51,15 @@ public String toString() {
 
     protected final String id;
     protected final Stream stream;
-    protected final Type type;
+    protected final String type;
```

Comment:
I would like to stick with it for now. It is used for legacy (builtin) alert conditions and removing it would provide no benefit to us currently.

I would vote for removing it later, when we go the extra mile of creating a migration job which transforms builtin alert condition types of persisted alert conditions to their full class name and we are handling their type just like all others.